pipeline/frontend/yaml/linter/schema/schema.json

```diff
@@ -212,6 +212,12 @@
         "image": {
           "$ref": "#/definitions/step_image"
         },
+        "privileged": {
+          "description": "Run the step in privileged mode. Read more: https://woodpecker-ci.org/docs/usage/pipeline-syntax#privileged",
+          "type": "boolean",
```

Comment:
Here should be reference only.
Put your definition on line around 431, between `step_image` and `step_pull`.
Look at `step_image`, for example.